apps/vc-api/src/main.ts

```diff
@@ -35,4 +35,4 @@ async function bootstrap() {
   }
 }
 
-bootstrap();
+void bootstrap();
```

Comment:
I have added the ESLint rule `typescript-eslint/no-floating-promises` to force handling promises to prevent it from reporting unhandled promise here. It is easy to make a mistake. Example error was fixed here: https://github.com/energywebfoundation/ssi/commit/186ad78c

Reply:
Ok, makes sense thanks